tsconfig.json

```diff
@@ -15,15 +15,18 @@
     "diagnostics": true
   },
   "include": [
+    // TODO(bckenny): unnecessary workaround until https://github.com/Microsoft/TypeScript/issues/24062
+    "node_modules/@types/node/index.d.ts",
```

Comment:
annoying, hopefully temporary issue where the compiler things that `EventEmitter` has a duplicate identifier. Manually putting the `node` types first fixes this for now.